### PR TITLE
v0x05/symmetric/hello

### PR DESCRIPTION
**Pull request by RoyAl82**
This is the file of the feature hello messages for v0x05. There are some imports that are missing because they are in another files that will be added later.

### CODE REVIEW

File: pyof/v0x05/common/header.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+# Classes
+
+class Header(GenericStruct):
+    """Representation of an OpenFlow message Header."""
+
+    version = UBInt8(OFP_VERSION)               # OFP_VERSION
+    message_type = UBInt8(enum_ref=Type)        # One of the OFPT_ constants
+    length = UBInt16()                          # Length including this ofp_header
+    xid = UBInt32()                             # Transaction id associated with this packet.
+                                                # Replies use the same id as was in the request to faciliate pairing.
```
Comment (hdiogenes) [inline pyof/v0x05/common/header.py]: All the comments here are duplicated in the `__init__` docstring, so it's better to just remove the comments.

Comment (RoyAl82) [reply pyof/v0x05/common/header.py]: I deleted those.

**Comment by RoyAl82** [comment]
Why can't I make more commits to the branch? It says that the commit to the branch (push) was rejected.

**Comment by RoyAl82** [comment]
I don't know what missing blank line is need it. Could you help me? I did the test in my computer and it passed it all. I don't know why it doesn't pass it in here. Thanks in advance.

**Pull request closed without merging**